cookbooks/calico/recipes/control.rb

```diff
@@ -372,12 +383,29 @@
     notifies :create, "template[/etc/nova/nova.conf]", :immediately
     notifies :run, "execute[remove-old-nova-db]", :immediately
 end
+template "/etc/nova/nova.conf" do
+    mode "0640"
+    source "control/nova.conf.erb"
+    variables({
+        admin_password: node[:calico][:admin_password],
+        live_migreate: node[:calico][:live_migrate]
+    })
+    owner "nova"
+    group "nova"
+    notifies :install, "package[nfs-kernel-server]", :immediately
```

Comment:
Are these triggered by the `:live_migrate` flag? I can't infer that from the structure.